lib/halfnarp.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
```

Comment:
well we have halfnarp-data for 32c3 and we will have it for 34c3. Also it is not good practice to hardcode event-names into a software, that is usable and might be used by anyone (isn't this the reason we renamed it from media.ccc.de to voctoweb?)